files/server/pfcat

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# pfcat - Helper script to print out mail from the Postfix queues.
+#
+# usage: pfcat <postfix queue id>
+#
+
+PATH=/usr/bin:/usr/sbin
+QS="deferred active incoming maildrop hold"
+QPATH=`postconf -h queue_directory`
+
+if [ $# -ne 1 ]; then
+	echo "Usage: pfcat <queue id>"
+	exit 1
+fi
+
+if [ `whoami` != "root" ]; then
+	echo "You must be root to view queue files."
+	exit 1
+fi
+
+if [ ! -d $QPATH ]; then
+	echo "Cannot locate queue directory $QPATH."
+	exit 1
+fi
+
+for q in $QS
+do
+	FILE=`find $QPATH/$q -type f -name $1`
+	if [ -n "$FILE" ]; then
+		postcat $FILE
+		exit 0
+	fi
+done
+
+if [ -z $FILE ]; then
+	echo "No such queue file $1"
+	exit 1
+fi
```

Comment:
Can you please run `shellcheck` on this and fix the issues? Also please convert all tabs to spaces.